src/main/java/appeng/bootstrap/FeatureFactory.java

```diff
@@ -123,39 +127,43 @@ public FeatureFactory features( AEFeature... features )
 		return new FeatureFactory( this, features );
 	}
 
-	void addBootstrapComponent( IBootstrapComponent component )
+	public <T extends IBootstrapComponent> void addBootstrapComponent( T component )
```

Comment:
Isn't this needed that java knows what functional interface to use for the lambdas?